app/javascript/packs/admin/editUser.jsx

```diff
@@ -115,3 +117,28 @@ const openModal = async (event) => {
 };
 
 document.body.addEventListener('click', openModal);
+
+const tagsRoot = document.getElementById('tag-moderation-container');
+
+// Normally, Preact can diff the server-side rendered DOM to hydrate the client-side rendered DOM,
+// but in this case, the MultiSelectAutocomplete component has a lot of markup, so I opted for
+// just a stylized input which is why innerHTML is used.
+tagsRoot.innerHTML = '';
+
+const currentModTags = JSON.parse(
+  tagsRoot.closest('form')['current_mod_tags'].value,
+);
+
+render(
+  <MultiSelectAutocomplete
+    defaultValue={currentModTags}
+    fetchSuggestions={fetchTags}
+    labelText="Assign tags"
+    maxSelections={Math.MAX_VALUE}
```

Comment:
I fell silly putting `Math.MAX_VALUE` at the moment. This could probably be something internal if no `maxSelections` prop is set @aitchiss ?

Reply:
`maxSelections` is an optional prop, and we don't limit selections if it's not passed in. I think you can just omit the prop here 🙂